app/views/event_staging/events/app.html.erb

```diff
@@ -327,21 +331,24 @@
 										<span class="help-tooltip" tabindex="-1" data-toggle="static-popover" data-placement="bottom" data-content="Bacon ipsum dolor amet spare ribs cow prosciutto beef ribs venison. ">
 											<i class="glyphicon glyphicon-question-sign"></i>
 										</span>
-										<input type="text" class="form-control" v-model.number="model.latitude" v-mask="{ alias: 'decimal' }">
+										<input type="text" class="form-control" v-model.number="model.latitude" v-mask="{ alias: 'decimal', digits: 6 }">
 									</div>
 									<div class="form-group col-xs-4">
 										<label>Longitude</label>
 										<span class="help-tooltip" tabindex="-1" data-toggle="static-popover" data-placement="bottom" data-content="Bacon ipsum dolor amet spare ribs cow prosciutto beef ribs venison. ">
 											<i class="glyphicon glyphicon-question-sign"></i>
 										</span>
-										<input type="text" class="form-control" v-model.number="model.longitude" v-mask="{ alias: 'decimal' }">
+										<input type="text" class="form-control" v-model.number="model.longitude" v-mask="{ alias: 'decimal', digits: 6 }">
 									</div>
 									<div class="form-group col-xs-4">
 										<label>Elevation</label>
 										<span class="help-tooltip" tabindex="-1" data-toggle="static-popover" data-placement="bottom" data-content="Bacon ipsum dolor amet spare ribs cow prosciutto beef ribs venison. ">
 											<i class="glyphicon glyphicon-question-sign"></i>
 										</span>
-										<input type="text" class="form-control" v-model="model.elevation" v-mask="{ alias: 'integer' }">
+										<div class="input-group">
+											<input-units type="text" class="form-control" v-model.number="model.elevation" v-mask="{ alias: 'integer' }" :scale="units.forElevation()"/>
```

Comment:
remove / from /> for html5 and i don't know what this is, but is this part correct? :scale="units.forElevation()"